include/wayfire/util/duration.hpp

```diff
@@ -82,6 +82,11 @@ class duration_t
      */
     bool running();
 
+    /**
+     * Reverse the duration.
```

Comment:
This should also include a little bit more detailed description of what is going on, for ex. making progress go from 1 to 0, changing the time remaining of the duration.